gpii/node_modules/processReporter/src/ProcessReporter.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * GPII Process Reporter
+ *
+ * Copyright 2015 OCAD University
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("kettle", require);
+fluid.require("./processesBridge.js", require);
+
+fluid.defaults("gpii.processReporter", {
+    gradeNames: "fluid.component"
+});
+
+fluid.defaults("gpii.processReporter.base", {
+    gradeNames: ["kettle.app", "gpii.processReporter"],
+    requestHandlers: {
+        processGet: {
+            route: "/processes",
+            type: "gpii.processReporter.allSolutions.get",
+            method: "get"
+        },
+        processGetSingle: {
+            route: "/processes/:solutionID",
+            type: "gpii.processReporter.solution.get",
+            method: "get"
+        }
+    },
+    components: {
+        platformReporter: {
+            type: "gpii.platformReporter.native"
```

Comment:
@amb26 Actually, I don't think the PlatformReporter should be used by the ProcessReporter at all anymore.  It's information that the DeviceReporter reports, and the MatchMaker and/or FlowManager have access to it via the DR.

In fact, the PlatformReporter is [defined](https://github.com/GPII/universal/blob/master/gpii/node_modules/deviceReporter/src/DeviceReporter.js#L42) and [implemented](https://github.com/GPII/universal/blob/master/gpii/node_modules/deviceReporter/src/DeviceReporter.js#L165) within the DeviceReporter code.  An implication is that the DeviceReporter code should be modified, as you suggest, to make the "gpii.platformReporter.native" an abstract grade, use contextAwareness, and so on.

@javihernandez, what do you think?